_episodes/01-select.md

```diff
@@ -266,17 +266,15 @@ SeLeCt FaMiLy, PeRsOnAl FrOm PeRsOn;
 |Roerich |Valentina|
 |Danforth|Frank    |
 
-You can use SQL's case insensitivity to your advantage. For instance,
-some people choose to write SQL keywords (such as `SELECT` and `FROM`)
-in capital letters and **field** and **table** names in lower
-case. This can make it easier to locate parts of an SQL statement. For
-instance, you can scan the statement, quickly locate the prominent
-`FROM` keyword and know the table name follows.  Whatever casing
+You can use SQL's case insensitivity to your advantage, by making it easier
```

Comment:
Thanks @jamesscottbrown, what do you think about rephrasing the statements a bit. 

You can use SQL's case insensitivity to your advantage, `by making it easier`
**to**
by making it easier >> to make it easier to distinguish between the different
**or** 
by making it easier >>  to distinguish between the different